### PR TITLE
Update use-microsoft-365-cdn-with-spo.md

### PR DESCRIPTION
**Pull request by msbemba**
to reduce ambiguity removed the Germany cloud as it has been retired - https://www.microsoft.com/en-us/cloud-platform/germany-cloud-regions#:~:text=A%3A%20On%20October%2029%2C%202021,in%20Europe%20before%20this%20time.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8941

File: microsoft-365/enterprise/use-microsoft-365-cdn-with-spo.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ description: Learn how to use the Office 365 Content Delivery Network (CDN) to s
 You can use the built-in Office 365 Content Delivery Network (CDN) to host static assets to provide better performance for your SharePoint Online pages. The Office 365 CDN improves performance by caching static assets closer to the browsers requesting them, which helps to speed up downloads and reduce latency. Also, the Office 365 CDN uses the [HTTP/2 protocol](https://en.wikipedia.org/wiki/HTTP/2) for improved compression and HTTP pipelining. The Office 365 CDN service is included as part of your SharePoint Online subscription.
 
 > [!NOTE]
-> The Office 365 CDN is only available to tenants in the **Production** (worldwide) cloud. Tenants in the US Government, China and Germany clouds do not currently support the Office 365 CDN.
+> The Office 365 CDN is only available to tenants in the **Production** (worldwide) cloud. Tenants in the US Government amd China clouds do not currently support the Office 365 CDN.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/use-microsoft-365-cdn-with-spo.md]:
```suggestion
> The Office 365 CDN is only available to tenants in the **Production** (worldwide) cloud. Tenants in the US Government and China clouds do not currently support the Office 365 CDN.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**